bsdlib/doc/security_tags.rst

```diff
@@ -0,0 +1,68 @@
+.. _security_tags:
+
+Security tags
+#############
+
+The modem on an nRF9160 device is equipped with a full IPv4/IPv6 stack with DTLS/TLS support.
+To use the cryptographic functions in the modem, the application must provision the security credentials to the modem.
+
+To be able to provision credentials, the modem must be offline or in offline mode.
+The credentials are provisioned through AT commands (see `Credential storage management %CMNG <https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/cmng.html>`_).
+If your device is running an nRF Connect SDK application, you can also use LTE Link Monitor to `manage the credentials <https://infocenter.nordicsemi.com/topic/ug_link_monitor/UG/link_monitor/lm_certificate_manager.html>`_.
```

Comment:
On line 68 we say something similar, though one is a `note` at the end of the document, the other is here. Perhaps we could combine this information.

Reply:
True - I moved the content from the note up and rephrased this paragraph.